.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "semi": false,
+  "printWidth": 100
+}
```

Comment:
I think I will opt for the configuration we used in https://github.com/keystonejs/keystone/pull/8910